### DISCUSSION TITLE
Data Safety

### DISCUSSION THREAD

**Discussion by ghost** [TeselaGen/openVectorEditor#877] [Q&A] [answered]
Hi there,

first of all, I just wanted to mention that I love OVE. I just stumbled upon it by chance via alternativo.net when I was looking for a cloning software to use after migrating to linux.

I have no experience with coding so far so I am not able to look for an answer within the code, and I did not find any information about this, so I would like to ask if the nucleotide data stays on the local computer, or if it is somehow send and analyzed, e.g. for improving the software? Nowadays, data security is a quite important feature when deciding for a software to use, especially if the the project might turn to a patent application.

I am sorry, I hope this is not a rude question.

**Comment by tnrich** [comment, marked as answer]
Hi @Fergna great question, the OVE package is not connected to any external APIs or sending data to any external entities. All data stays local to the machine it is running on

**Reply by ghost** [reply]
Hey @tnrich thank you very much! That makes OVE even more attractive!
